packages/core/src/app/shipping/MultiShippingFormV2.test.tsx

```diff
@@ -275,4 +275,53 @@ describe('MultiShippingFormV2 Component', () => {
 
         expect(screen.getByText('All items are allocated.')).toBeInTheDocument();
     });
+
+    it('removes a consignment', async () => {
+        jest.spyOn(checkoutService, 'deleteConsignment').mockReturnValue(
+            Promise.resolve(checkoutService.getState()),
+        );
+        
+        jest.spyOn(checkoutState.data, 'getCheckout').mockReturnValue({
+            ...getCheckout(),
+            consignments: [{
+                ...getConsignment(),
+                lineItemIds: [
+                    getPhysicalItem().id.toString(),
+                ]
+            }],
+            cart: {
+                ...getCart(),
+                lineItems: {
+                    ...getCart().lineItems,
+                    physicalItems: [{
+                        ...getPhysicalItem(),
+                        quantity: 2,
+                    },
+                    {
+                        ...getPhysicalItem(),
+                        id: '2',
+                        quantity: 1,
+                    }],
+                },
+            },
+        });
+
+        render(
+            <CheckoutProvider checkoutService={checkoutService}>
+                <LocaleContext.Provider value={localeContext}>
+                    <ExtensionProvider checkoutService={checkoutService}>
+                        <MultiShippingFormV2 {...defaultProps} />
+                    </ExtensionProvider>
+                </LocaleContext.Provider>
+            </CheckoutProvider>,
+        );
+
+        const closeButton = screen.getByTestId('consignment-close-button');
+
+        expect(closeButton).toBeInTheDocument();
+
+        await userEvent.click(closeButton);
+
+        expect(checkoutService.deleteConsignment).toHaveBeenCalledWith(getConsignment().id);
```

Comment:
We might need to discuss this further.
I feel we need to assert UI changes mroe.
cc @richapatel1510